### PR TITLE
Update FAQ to reflect Token Launch (#2540)

### PR DESCRIPTION
**Pull request by anxolin**
Apply this PR https://github.com/gnosis/cowswap/pull/2540 as a cherry-pick

### CODE REVIEW

**Comment by elena-zh** [comment]
Changes LGTM!
But still 1 period is missing in the 'What is the purpose of COW Token?' section
image

**Comment by elena-zh** [comment]
Maybe it was mentioned before, but I'd make these categories in bold:
image

**Review by W3stside — APPROVED** [review]

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -573,6 +556,102 @@ export default function Faq() {
...
+          <p>
+            CoW POAP: Awarded POAPs from the testing sessions, Batch challenges, CowStars or user research sessions. Not
+            all are equally valuable. Some that are most special are the ones for our CowStars and User Test
+            Participants. Note that we say “awarded” and not “owns”, since we know that there have been secondary
+            markets for these POAPS. We only took awardees into consideration.
+          </p>
+          <p>
+            Power Traders: A Power trader is someone who has traded on either Ethereum mainnet (before block 13974427)
+            or Gnosis Chain (block 20024195) a minimum of 3 trades, at least 14 days between the 1st and the last trade,
+            and for a minimum of 1K total volume. Note that stable to stable coin trades only by factor 0.1
```
Comment (nenadV91) [inline src/custom/pages/Faq/index.tsx:638]: Mssing `.` at the end here

File: src/custom/pages/Faq/index.tsx [hunk 2]
```diff
@@ -573,6 +556,102 @@ export default function Faq() {
...
+            Traders: A trader is someone who has traded on either Ethereum mainnet (before block 13974427) or Gnosis
+            Chain (block 20024195) at least once for a total of 1K USD or who has done at least 5 trades. Note that
+            stable to stable coin trades only by factor 0.1.
+          </p>
+          <p>
+            GNO Holders: A GNO holder is someone who held at least 0.1 GNO on either Ethereum mainnet (before block
+            13974427) or Gnosis Chain (block 20024195). They were eligible for holding GNO on Mainnet and/or
+            GnosisChain, and for running GBC validators. In addition, all the following LP token holders were accounted
+            for: Balancer v2 (mainnet), Uniswap v3 (mainnet), Honeyswap (gnosischain), Symetric (gnosischain), Sushiswap
+            (gnosischain), Elk (gnosischain), Swapr (Gnosischain).
```
Comment (nenadV91) [inline src/custom/pages/Faq/index.tsx:650]: Maybe we should use lower case on the last `(Gnosischain).` for consistency :smile:

**Review by nenadV91 — APPROVED** [review]
Apart from the mentioned comments and build issue, looks good to me :+1:

**Comment by anxolin** [comment]
Done 
image

**Comment by anxolin** [comment]
Done 

image

**Comment by anxolin** [comment]
Merging, to prepare release to not delay it more. 


Any new comment, we can apply to the one @fairlighteth is preparing

**Comment by elena-zh** [comment]
@anxolin , nice!
I have asked @fairlighteth to fix this in his PR: 

> in this paragraph we have different writing of networks names. Some are started with the capital letter, some with lowercase ones. Gnosis chain is spelled somewhere together, somewhere separately. I think, we should be consistent with networks spelling
image

**Pull request merged**